numpy/core/_add_newdocs.py

```diff
@@ -3114,13 +3114,19 @@
         'C' otherwise. 'K' means match the layout of `a` as closely
         as possible. (Note that this function and :func:`numpy.copy` are very
         similar, but have different default values for their order=
```

Comment:
nit: Because of the added text at the end of the sentence, I suggest
```suggestion
        as possible. (Note that this function and :func:`numpy.copy` are very
        similar but have different default values for their order=
```